repository/repository.go

```diff
@@ -470,3 +476,169 @@ To find more information on specific steps, please see documentation [here](http
 - [ ] QA: Final validation of above PR and tracked through the linked ticket
 - [ ] PJM: Close the milestone in GitHub.
 `
+
+// GetUpstreamRemote will return the remote name for a given configured remote URL
+func GetUpstreamRemote(r *git.Repository, rURL string) (string, error) {
```

Comment:
Let's remove "Get" from this function and call it "UpstramRemote". "Get" is a smell in Go.